src/map/battle.cpp

```diff
@@ -9630,7 +9630,7 @@ struct Damage battle_calc_misc_attack(struct block_list *src,struct block_list *
 		case BA_DISSONANCE:
 			md.damage = 30 + skill_lv * 10;
 			if (sd)
-				md.damage += 3 * pc_checkskill(sd,BA_MUSICALLESSON);
+				md.damage += skill_lv * pc_checkskill(sd,BA_MUSICALLESSON);
```

Comment:
`if (sd)` could possibly be removed since sd check is redundant ; pc_checkskill returns 0 when sd is nullptr
```
uint8 pc_checkskill(map_session_data *sd, uint16 skill_id)
{
	uint16 idx = 0;
	if (sd == nullptr)
		return 0;
```

Reply:
That makes it much easier! Will improve that then.